package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.8
+FROM alpine:3.12
```

Comment:
Once this PR is merged, the commit will be tagged with `v0.2.1` instead of `v0.3.0` since there were no code changes introduced.

Reviewers: can you thumbs up if this sounds good to you (or comment if not)?